admin/client/src/components/Preview/Preview.js

```diff
@@ -20,7 +20,11 @@ class Preview extends SilverStripeComponent {
         <div className="toolbar--south">
           <div className="btn-toolbar">
             <button className="btn btn-secondary btn-secondary" type="button"><i className="font-icon-edit"></i> Edit</button>
-            <button type="button" data-container="body" className="btn btn-link btn--options" data-toggle="popover" title="Page actions" data-placement="top" data-content="<a href=''>Add to campaign</a><a href=''>Remove from campaign</a>"><i className="font-icon-dot-3"></i></button>
+            {/* More options button and popover for Alpha 2
```

Comment:
We need to ensure the work that has gone into creating this markup (incl. details like setting a tabindex) doesn't get lost. I'm fine with commented out code as long as there's a clear path to enabling it again, which there is in this case since we'll need tie feature soon. If removing it, please create a Github issue which contains the markup as a starting point.